src/utils/displayFunctions.ts

```diff
@@ -8,6 +8,11 @@ import { IST_ICON } from 'assets/assets';
 import type { BrandInfo } from 'store/app';
 import { wellKnownBrands } from 'config';
 
+export const notifyError = err => {
+  console.log(err);
+  window.alert(err);
```

Comment:
Not alert(err.message)?

Reply:
That would omit "Error: ".
<img width="456" alt="Screen Shot 2022-11-04 at 7 36 09 AM" src="https://user-images.githubusercontent.com/21505/200001592-2547611e-b3d8-47c2-97bb-791f11a61c3c.png">

Reply:
ok. I was worried about `[Object object]` or some such. thanks for the detail.